### PR TITLE
add button, aligment content

### PR DESCRIPTION
**Pull request by eduard-terletskiy**
добавил кнопку и сделал отступы

### CODE REVIEW

File: corn/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700&display=swap" rel="stylesheet">
   <!-- If you uncomment the following line the design MUST be dark -->
-  <!--link rel="stylesheet" href="styles.black.css"-->
+  <!-link rel="stylesheet" href="styles.black.css" -->
```
Comment (glebhg) [inline corn/index.html]:
```suggestion
  <!--link rel="stylesheet" href="styles.black.css" -->
```

File: corn/styles.css [hunk 2]
```diff
@@ -1,70 +1,134 @@
...
+  --padding-inline-start-ul: 0px;
+  --margin-right-input: 20px;
+  --margin-left-input: 0;
+  --margin-bottom-h2: 0;
+  --margin-top-h2: 20px;
+  --padding-bottom-ul-complete_list-li: 10px;
+  --display-button: block;
+  --position--button: sticky;
+  --bottom-button: 0;
+}
```
Comment (glebhg) [inline corn/styles.css:27]: I guess you don't need so many custom properties.

File: corn/styles.css [hunk 3]
```diff
@@ -1,70 +1,134 @@
...
-    --font-size-html:18px;
-    --font-size-p: 14px;
-    --font-size-h1:32px;
-    --font-weight-h1-h2:700;
-    --font-weight-p:600;
-    --font-weight-li:500;
-    --color-p-label:#B9B9BE;
-    
-    
+:root {
```
Comment (glebhg) [inline corn/styles.css:1]: Looks like it positioned wrongly:

Screenshot 2021-06-28 at 16 42 21

**Review by glebhg — CHANGES_REQUESTED** [review]

File: corn/styles.css (hunk 2 above)
Comment (glebhg) [reply corn/styles.css:27]: You just need here properties that you will override in `styles.black.css`

**Review by glebhg — CHANGES_REQUESTED** [review]

**Review by yogahub — COMMENTED** [review]
All fixed....

**Pull request closed without merging**